infoschema/tables_test.go

```diff
@@ -543,7 +543,6 @@ func (s *testTableSuite) TestSomeTables(c *C) {
 		Command: byte(1),
 		Digest:  "abc1",
 		State:   1,
-		StmtCtx: tk.Se.GetSessionVars().StmtCtx,
```

Comment:
Take out `tk.Se.GetSessionVars().StmtCtx` for future use is not correct, it's not a snapshot.
The correct usage should be Lock -> Read data -> Unlock for process info ...
See https://github.com/pingcap/tidb/pull/26196